src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+import { getMovie } from '../../api/api';
+
+export const FindMovie = ({ addMovie }) => {
+  const [movie, setMovie] = useState([]);
+  const [title, setTitle] = useState('');
+  const [error, setError] = useState(false);
+
+  const movieSearch = (name) => {
+    getMovie(name)
+      .then((data) => {
+        setMovie({
+          title: data.Title,
+          description: data.Plot,
+          imgUrl: data.Poster,
+          imdbUrl: `https://www.imdb.com/title/${data.imdbID}/`,
+          imdbId: data.imdbID,
+        });
```

Comment:
I think will better take normalize data in your API file, because if server system will change you must solve mistake in a lot of places

Reply:
Don't agree here, api usually doesn't know in what format do you need data on frontend. You get what you get and then have to work with it.